lib/devise/models/confirmable.rb

```diff
@@ -169,7 +175,6 @@ def skip_reconfirmation!
         # in models to map to a nice sign up e-mail.
         def send_on_create_confirmation_instructions
           send_confirmation_instructions
-          skip_reconfirmation!
```

Comment:
I'm not sure if we can remove this line. It was introduced in this PR https://github.com/plataformatec/devise/pull/4043/files